projects/flyout/src/test.ts

```diff
@@ -26,3 +26,9 @@ getTestBed().initTestEnvironment(
 const context = require.context('./', true, /\.spec\.ts$/);
 // And load the modules.
 context.keys().map(context);
+
+// Find any tests included in the "testing" entry point.
+try {
+  const testingContext = require.context('../testing/', true, /\.spec\.ts$/);
+  testingContext.keys().map(testingContext);
```

Comment:
You might see code coverage drop. If so, add `/*istanbul ignore next*/` above this line.